### ISSUE TITLE
Colombian Request Channels

### ISSUE THREAD

**Issue by freearhey** [iptv-org/iptv#3065]
List of requests for Colombian channels from other issues:

- [x] [Caracol TV](https://www.caracoltv.com/senal-vivo) (#2893)
- [x] [Canal 1](https://canal1.com.co/senal-en-vivo) (#2890)
- [ ] TV5 Montería (#4194)
- [ ] Zoom (#4233)

If you find a working link to a channel from the list you can add it directly to the playlist by making a pull request, just read this [guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#add-or-replace-a-stream) before you do it.

**Comment by freearhey** [comment]
@jahirfiquitiva Isn't that the channel you asked for?
https://github.com/iptv-org/iptv/blob/2b781967d600add3d7d255389c8a333e8bcfd35a/channels/co.m3u#L28-L29

**Comment by LaneSh4d0w** [comment]
Noticias Caracol and Caracol TV are two different channels.

**Comment by freearhey** [comment]
@LaneSh4d0w Thanks, I'll keep that in mind.

**Comment by LaneSh4d0w** [comment]
OK I have a "Canal Caracol" thing is I have no video and no audio, but VLC thinks i'm streaming something.

Can anyone verify this : http://45.179.140.242:8000/play/a0hn

**Comment by Dum4G** [comment]
> http://45.179.140.242:8000/play/a0hn

ERR_CONNECTION_TIMED_OUT

**Comment by LaneSh4d0w** [comment]
> > http://45.179.140.242:8000/play/a0hn
> 
> ERR_CONNECTION_TIMED_OUT

Yeah, I see the same result now. I'm going to check the links again.

**Comment by LaneSh4d0w** [comment]
![image](https://user-images.githubusercontent.com/30985701/121772886-9cac8c80-cb78-11eb-9c73-7b94ae86b912.png)

It came back, programme's encrypted.

**Comment by LaneSh4d0w** [comment]
@jahirfiquitiva Test this one : http://186.101.216.211:8000/play/a08z

**Comment by jahirfiquitiva** [comment]
@LaneSh4d0w that one works 👍

**Comment by LaneSh4d0w** [comment]
> 
> 
> @LaneSh4d0w that one works 👍

Wonderful. Adding it.

**Comment by LaneSh4d0w** [comment]
@jahirfiquitiva @freearhey Last draft PR i'm curating right now should contain both Caracol and Canal 1. I'm adding more channels at the same instance, so you'll wait a bit until I can confirm everything's good.
